spec/services/group_finalizer_spec.rb

```diff
@@ -12,9 +12,18 @@
           .and_return(instance_spy('Draw', open_suite_sizes: [g.size]))
         membership = instance_spy('Membership', locked: false)
         leader = instance_spy('User', membership: membership)
+        m = Array.new(g.size - 1) { |_| instance_spy('user') }
```

Comment:
Whew this let block is getting ugly - maybe we extract to a `mock_group` method?